src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -88,6 +88,7 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction First = _library.Append(new FirstLastFunction(isFirst: true));
         public static readonly TexlFunction FirstN = _library.Append(new FirstLastNFunction(isFirst: true));
         public static readonly TexlFunction ForAll = _library.Append(new ForAllFunction());
+        public static readonly TexlFunction ForAll_UO = _library.Append(new ForAllFunction_UO());
```

Comment:
>ForAll_UO [](http://example.com/codeflow?start=44&length=9)

Adding this to the core library will require changes in Power Apps; make sure to only complete this PR once it's ready to be migrated to the other repository

Reply:
I added the Breaking Internals label.

Reply:
We can keep this PR open until the PA client change is working.

Reply:
PA client change is looking very good right now.